guides/common/modules/proc_configuring-a-physical-interface.adoc

```diff
@@ -0,0 +1,37 @@
+[id="configuring-a-physical-interface"]
+= Configuring a physical interface
+
+You can configure a physical interface for a host.
+
+.Procedure
```

Comment:
I am concerned about the number of subprocedures you are creating. The general interface, ntwork, and DNS settings could be converted to bullet lists.

I'm not sure navigating to "Add Interface" should be a subprocedure. I would make them part of the procedure.